src/elements/core/overlay/inert.ts

```diff
@@ -0,0 +1,108 @@
+const IGNORED_ELEMENTS = ['script', 'head', 'template'];
```

Comment:
is there maybe more elements which should be ignored? Maybe <link/> <style/> <meta/> ? Are we assuming here that those elements are usually inside the <head/> element?

Reply:
Thanks for the hint, we can add style as well. These elements in the list have no direct impact on the inert mechanism as they are more meta-like. Conversely, there wouldn't be any change if they received inert, so at the end it doesn't change much.